sdv/data_processing/data_processor.py

```diff
@@ -537,9 +537,10 @@ def update_transformers(self, column_name_to_transformer):
                     "'RegexGenerator' instead."
                 )
 
-        warnings.filterwarnings('ignore', module='rdt')
-        self._hyper_transformer.update_transformers(column_name_to_transformer)
-        warnings.resetwarnings()
+        with warnings.catch_warnings():
+            msg = rdt.HyperTransformer._REFIT_MESSAGE
```

Comment:
Can we just filter all warnings from the `HyperTransformer`? Seems like thats what we were doing before (actually more broad since it filtered all of RDT) and that way we don't have to worry about the message